lang/java/avro/src/test/java/org/apache/avro/TestSchema.java

```diff
@@ -177,4 +178,24 @@ private Schema createDefaultRecord() {
     return Schema.createRecord("name", "doc", "namespace", false);
   }
 
+  @Test
+  public void testSerialization() throws IOException, ClassNotFoundException {
+    ByteArrayOutputStream bos = new ByteArrayOutputStream();
```

Comment:
Can you wrap these with `try-with-resources`. I know what it is just a test, but I like to give a good example 👍

Reply:
Sounds good -- I'll also see if there's an existing schema we can reuse for testing serialization.